app/assets/javascripts/pdfjs_viewer/viewer.js

```diff
@@ -7144,6 +7144,7 @@ window.PDFView = PDFViewerApplication; // obsolete name, using it as an alias
 
 var HOSTED_VIEWER_ORIGINS = ['null',
   'http://mozilla.github.io', 'https://mozilla.github.io'];
+var DISABLE_RANGE = false;
```

Comment:
If the normal default is true, we should have this also default to true.  Then in our code we will override to false